### ISSUE TITLE
swin transformer

### ISSUE THREAD

**Issue by shuyuan-wang** [megvii-research/Sparsebit#119]
is this working for swin transformer?

**Comment by PeiqinSun** [comment]
Yes, Sparsebit is a framework which can adapt to any CNNs and Transformers in theory. But you maybe need add some QModules by yourself. Welcome to raise an issue after you try it.
